build/layerutils/manifests/manifests.go

```diff
@@ -35,13 +36,14 @@ func (manifests *Manifests) ListImages(yamlFile string) ([]string, error) {
 
 	images := layerutils.DecodeImages(string(yamlBytes))
 	if len(images) != 0 {
-		list = append(list, images...)
+		for _, image := range images {
+			named, err := reference.ParseToNamed(image)
```

Comment:
1. no need return err, just continue 
2. image validation should be applied for all image list searched from build context.